chalice/constants.py

```diff
@@ -70,3 +70,142 @@ def index():
     ],
     "Resource": "arn:aws:logs:*:*:*"
 }
+
+
+CODEBUILD_POLICY = {
+    "Version": "2012-10-17",
+    # This is the policy straight from the console.
+    "Statement": [
+        {
+            "Action": [
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream",
+                "logs:PutLogEvents"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "s3:GetObject",
+                "s3:GetObjectVersion",
+                "s3:PutObject"
+            ],
+            "Resource": "arn:aws:s3:::*",
+            "Effect": "Allow"
+        }
+    ]
+}
+
+CODEPIPELINE_POLICY = {
```

Comment:
Yeah. I am fine keeping it as it. It just pointed it out as it looked weird given you really only used half of those services and normally for IAM policies you keep it as minimal as possible.